JPS_Ontology/ontology/ontoems/OntoEMSExtension.csv

```diff
@@ -0,0 +1,107 @@
+AtmosphericPressure,Class,http://purl.org/iot/vocab/m3-lite#AtmosphericPressure,EQUIVALENT-TO,,,,The pressure exerted by the atmospheric column of air above the point in question (adjusted to mean sea level).,https://www.theworldavatar.com/kg/ontoems/,Atmospheric pressure
+AtmosphericPressure,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#AtmosphericPressure,EQUIVALENT-TO,,,,The pressure exerted by the atmospheric column of air above the point in question (adjusted to mean sea level).,https://www.theworldavatar.com/kg/ontoems/,Atmospheric pressure
+RelativeHumidity,Class,http://purl.org/iot/vocab/m3-lite#RelativeHumidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+RelativeHumidity,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#Humidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+RelativeHumidity,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/RelativeHumidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+Visibility,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Distance,IS-A,,,,"Prevailing horizontal distance at which an object can be clearly discerned by a certified weather observer, under current weather conditions.",https://www.theworldavatar.com/kg/ontoems/,Visibility
+UVIndex,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The strength of the sun's ultraviolet (UV) radiation expressed as index as developed by the World Health Organization.,https://www.theworldavatar.com/kg/ontoems/,UV index
+PrecipitationProbability,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
+PrecipitationProbability,Class,http://semanticscience.org/resource/SIO_000638,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
+AirPollutantConcentration,Class,https://www.theworldavatar.com/kg/ontouom/Concentration,IS-A,,,,The concentration of a specific air pollutant as amount of material per unit volume of air (e.g. mass concentration).,https://www.theworldavatar.com/kg/ontoems/,Air pollutant concentration
+http://www.ontology-of-units-of-measure.org/resource/om-2/VolumeFraction,Class,https://www.theworldavatar.com/kg/ontouom/Concentration,IS-A,,,,,https://www.theworldavatar.com/kg/ontoems/,Volume fraction
+https://www.theworldavatar.com/kg/ontouom/MassOfSubstanceConcentration,Class,https://www.theworldavatar.com/kg/ontouom/Concentration,IS-A,,,,,https://www.theworldavatar.com/kg/ontouom/,Mass concentration
```

Comment:
There are quite a few definitions in this CSV file of things that are defined elsewhere, e.g. OM, OntoTimeSeries, and the environment agency flood monitoring ontology. Please don't repeat other people's definitions.

Reply:
> There are quite a few definitions in this CSV file of things that are defined elsewhere, e.g. OM, OntoTimeSeries, and the environment agency flood monitoring ontology. Please don't repeat other people's definitions.

Isn't this what the "Defined By" column is for in the .csv template? So, one can keep all relevant concepts and relationships together (mainly for overview and clarify) without re-inventing them. That was my understanding when creating the initial OntoEMS version with Feroz

Reply:
I believe repeating other people's definitions is a bad idea as a matter of principle. If the original definitions change, then your 'repetition' becomes out-of-sync, i.e. a contradiction to the (revised) original. For that reason, I'm convinced it is best to avoid this.